src/item.cpp

```diff
@@ -141,7 +142,17 @@ Item* Item::CreateItem(PropStream& propStream)
 			break;
 	}
 
-	return Item::CreateItem(id, 0);
+    const ItemType& iType = items[id];
+    unsigned char count = 0;
+    if(mapVersion == 0) {
```

Comment:
Fix coding style, uchar to uint8 and all ifs to match tfs syntax

Reply:
unsigned char replaced by uint8_t and adding spaces between "if" and "(" as well as added space between ")" and "{"